securedrop/journalist_app/__init__.py

```diff
@@ -18,15 +18,21 @@
 from models import Journalist
 from store import Storage
 
+MYPY = False
+if MYPY:
+    from sdconfig import SDConfig
+
 _insecure_views = ['main.login', 'static']
 
 
 def create_app(config):
+    # type: (SDConfig) -> Flask
     app = Flask(__name__,
                 template_folder=config.JOURNALIST_TEMPLATES_DIR,
                 static_folder=path.join(config.SECUREDROP_ROOT, 'static'))
 
     app.config.from_object(config.JournalistInterfaceFlaskConfig)
+    app._config = config
```

Comment:
We should assign this to `sdconfig` because I feel like that is more pythonic since accessing "protected" members implies monkey-patchiness.

Reply:
Can we fix this in a future PR?

Reply:
We already renamed everything in this PR so I feel like we might as well rename it right the first time if we agree that it makes more sense to name it `sdconfig`.